src/core/server/saved_objects/export/get_sorted_objects_for_export.ts

```diff
@@ -66,18 +66,21 @@ async function fetchObjectsToExport({
       throw err;
     }
     return bulkGetResult.saved_objects;
+  } else if (types && types.length > 0) {
+    const findResponse = await savedObjectsClient.find({
+      type: types,
+      sortField: '_id',
+      sortOrder: 'asc',
+      perPage: exportSizeLimit,
+      namespace,
+    });
+    if (findResponse.total > exportSizeLimit) {
+      throw Boom.badRequest(`Can't export more than ${exportSizeLimit} objects`);
+    }
+    return findResponse.saved_objects;
+  } else {
+    throw Boom.badRequest('Either `type` or `objects` are required.');
```

Comment:
This case is also handled by the Joi schema validation at the route handler. But adding it here makes TS happy and the code more defensive.

Reply:
Is there existing test coverage for this? I don't see any new tests added in this file.